src/cronSelection.js

```diff
@@ -1,59 +1,59 @@
-'use strict';
+"use strict";
 
-angular.module('angular-cron-jobs', ['templates-angularcronjobs']);
+angular.module("angular-cron-jobs", ["templates-angularcronjobs"]);
 
-angular.module('angular-cron-jobs').directive('cronSelection', ['cronService', function(cronService) {
+angular.module("angular-cron-jobs").directive("cronSelection", ["cronService", function(cronService) {
     return {
-        restrict: 'EA',
+        restrict: "EA",
         replace: true,
         transclude: true,
-        require: 'ngModel',
+        require: "ngModel",
         scope: {
```

Comment:
Never mind I see you set it from config below. Are you referencing `cronStyle` in the view? If not lets avoid binding it to `$scope` since that will create more watchers that are unnecessary.